scripts/release/steps/bump-prettier.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const { exec } = require("child-process-promise");
+const { logPromise } = require("../utils");
+
+async function format() {
+  await exec("yarn lint --fix");
+  await exec("yarn lint-docs --fix");
+}
+
+async function commit(version) {
+  await exec(`git commit -am "Bump Prettier dependency to ${version}"`);
```

Comment:
Since you’re using `exec` here, someone could put a string like `" && rm -rf / && echo "` as `version` and run arbitrary code (not sure how dangerous this is).

Reply:
We're the ones who are gonna use this, so I guess it's on us to be careful...

In this specific case, the script checks if `version` is valid using semver.

Reply:
Why not use `spawn` instead?

```js
await spawn("git", ["commit", "-am", `Bump Prettier dependency to ${version}`])
```

Reply:
Why not use `spawn` instead?

```js
await spawn("git", ["commit", "-am", `Bump Prettier dependency to ${version}`])
```

Reply:
If you’re interested, I [created a simple `spawn` template tag](https://runkit.com/j-f/5b155a812ac4a80011ca80e2) that you can use so you don’t have to split the arguments individually.